feature-proxy-manager/src/main/java/com/kinandcarta/create/proxytoggle/manager/model/Proxy.kt

```diff
@@ -0,0 +1,30 @@
+package com.kinandcarta.create.proxytoggle.manager.model
+
+data class Proxy(val address: String, val port: Int) {
```

Comment:
I don't think this should be in the model package. A model doesn't contain any logic but this class does

Reply:
Extracted logic into `ProxyMapper` 👍